types/genesis.go

```diff
@@ -84,10 +84,6 @@ func (genDoc *GenesisDoc) ValidateAndComplete() error {
 		}
 	}
 
-	if genDoc.GenesisTime.IsZero() {
```

Comment:
why delete?

Reply:
I remember we needed this for an older version of Tendermint consensus that required synchrony.  We shouldn't need this anymore.